app/scripts/controllers/eventRegistrations.js

```diff
@@ -85,14 +86,34 @@ angular.module('confRegistrationWebApp')
       });
     }
 
-    // toggle (show/hide) column(s)
-    $scope.toggleColumn = function (block) {
+    //toggle (show/hide) columns
+    $scope.toggleColumn = function(block) {
       $scope.blocks[block].visible = !$scope.blocks[block].visible;
       visibleBlocks =  _.pluck(_.where($scope.blocks, { 'visible': true }), 'id');
       localStorage.setItem('visibleBlocks:' + conference.id, JSON.stringify(visibleBlocks));
       $scope.queryParameters.block = visibleBlocks;
     };
 
+    //turn on visible built in columns
```

Comment:
@OzzieOrca We have created a separate branch (https://github.com/CruGlobal/conf-registration-web/pull/497) which contains just the changes related to this branch. This has already been mentioned in one of the comments posted above, on last Friday. Please use this new branch for testing and discard this branch, as we would be pushing the changes to the new branch (https://github.com/CruGlobal/conf-registration-web/pull/497).